CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,140 @@
+
+# Code of Conduct
+
+## Our Pledge
+
+We as contributors pledge to make participation in our
+community a harassment-free experience for everyone, regardless of age, body
+size, visible or invisible disability, ethnicity, sex characteristics, gender
+identity and expression, level of experience, education, socio-economic status,
+nationality, personal appearance, race, caste, color, religion, or sexual
+identity and orientation.
```

Comment:
I don't think there's a benefit to trying to iterate every attribute irrelevant to development. Instead, mentioning at least some of these (which are in other contexts debatable whether they even exist) would infer a particular political position "of the project".

Suggest a generic "personal attributes or off-topic and irrelevant factors" or something like that.